build.xml

```diff
@@ -173,21 +188,12 @@
         <echo unless:set="prince-available" message="prince available: false"/>
 
         <antcall target="saxon-prepare" unless:true="${saxon-available}"/>
+        <antcall target="schematron-download" unless:true="${schematron-available}"/>
         <antcall target="xerces-download" unless:true="${xerces-available}"/>
 
         <echo>initialized</echo>
     </target>
 
-    <target name="init-mei-classpath" description="Initializes the mei.classpath, which is essential for the schema generation, by prepending the jar files contained in the lib directory to the java classpath. Will be called automatically if needed." depends="init">
-        <path id="mei.classpath">
-            <fileset dir="lib" erroronmissingdir="false">
-                <include name="**/*.jar" />
-            </fileset>
-            <pathelement path="${java.class.path}"/>
```

Comment:
I think deleting this was just for testing purposes… and maybe should stay

Reply:
Thanks for spotting! It was deleted unintentionally 🙈

Reply:
Works now in Docker, even when running `ant reset` first